test/unit/event/verification-method/hcs-did-update-verification-method-event.test.ts

```diff
@@ -0,0 +1,190 @@
+import { PrivateKey } from "@hashgraph/sdk";
+import { DidError, Hashing, HcsDidEventTargetName, HcsDidUpdateVerificationMethodEvent } from "../../../../dist";
+
+describe("HcsDidUpdateVerificationMethodEvent", () => {
+    const privateKey = PrivateKey.fromString(
+        "302e020100300506032b6570042204209044d8f201e4b0aa7ba8ed577b0334b8cb6e38aad6c596171b5b1246737f5079"
```

Comment:
hardcoded private key

Reply:
he value needs to be fixed to match the known output for the test cases.